docs/user_docs/merger_usage.md

```diff
@@ -0,0 +1,19 @@
+# Merger usage
+
+*Subject to change.*
+
+## Description
+
+The **Decentralized Tech Radar** has a Merger feature to combine the contents of multiple CSV spec-files.
+
+**Note:** The Merger does not currently handle duplicated content.
+
+## How to use
+
+When wanting to merge `file0.csv` and `file1.csv`, run the following command:
+
+```cmd
+go run main.go merge <path-to-file> <path-to-other-file>
```

Comment:
The command to run the merger needs to be:
`go run main.go -merge "<path-to-file> <path-to-other-file>"`

Reply:
Oh interesting, I could run it in Powershell without neither the hyphen nor encasing the path-files within quotation-marks. Might there be several ways of doing running the merger?

Reply:
That might be fine then. Will give it a try when I get home after work

Reply:
@August-Brandt I've added your command to the doc as well.

When you've got time: Did my command work on your machine? :)

Reply:
I have tried the `go run main.go merge <file path> <file path>` command now. It doesn't work in my terminal. At least not on my win 10 machine. It only works with `go run main.go -merge "<file path> <file path>"`

Reply:
I'll get Theis to test `go run main.go merge <path-to-file> <path-to-other-file>` on his Linux machine, and update the docs according to our findings! :)

*Update pending.*